### PR TITLE
Add container restart functionality

### PR DESCRIPTION
**Pull request by humenius**
Suggested in #1, this branch adds container restart functionalities to the Docker image.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,25 +1,59 @@
...
+	traefik-certs-dumper file --version v2 --crt-name "cert" --crt-ext ".pem" --key-name "key" --key-ext ".pem" --domain-subdir --dest /tmp/work --source /traefik/acme.json > /dev/null
+
+	if [[ -f /tmp/work/${DOMAIN}/cert.pem && -f /tmp/work/${DOMAIN}/key.pem && -f /output/cert.pem && -f /output/key.pem ]] && \
+		diff -q ${WORKDIR}/${DOMAIN}/cert.pem /output/cert.pem >/dev/null && \
+		diff -q ${WORKDIR}/${DOMAIN}/key.pem /output/key.pem >/dev/null ; \
+	then
+		log "Certificate and key still up to date, doing nothing"
+	else
+		log "Certificate or key differ, updating"
+		mv ${WORKDIR}/${DOMAIN}/*.pem /output/
```
Comment (maximbaz) [inline run.sh]: Thanks for adding this functionality 👍 I think you meant to call `restart_containers` after this line?

Comment (humenius) [reply run.sh]: Yeah, that got slipped away. I'm sorry, I'll fix it right away.

File: run.sh [hunk 2]
```diff
@@ -1,25 +1,153 @@
...
+    
+    if [ ! -z "${CONTAINERS#}" ]; then
+      log "Trying to restart containers"
+      restart_containers
+    fi
+	fi
+}
+
+restart_containers() {
+  for i in "${CONTAINERS[@]}"; do
```
Comment (maximbaz) [inline run.sh]: not sure if you care, you have a mix of tabs and spaces, and different number of spaces sometimes 🙂

Comment (humenius) [reply run.sh]: Yeah, Sublime and VS Code seem to mess things up although I have set the space indent properly. I'll try to fix it.

Comment (humenius) [reply run.sh]: This one should be fixed now

**Pull request merged**